.github/workflows/chromatic.yml

```
@@ -10,6 +10,8 @@ on:
 
 jobs:
   chromatic-deployment:
+    # To run only on the main repository and not forks
+    if: github.repository == 'rad-ui/ui'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)